recipes/itolapi/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "itolapi" %}
+{% set version = "4.0.0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 161759384603664ff21ad6ae412a01549ce19436fc6959478d932d9b6c9915c4
+
+build:
+  noarch: generic
```

Comment:
This probably needs to be `noarch: python`

Reply:
makes sense most probably a pure python library without compilation